controllers/provider-aws/charts/provider-aws/values.yaml

```
@@ -19,6 +19,7 @@ disableControllers:
 disableWebhooks:
 - controlplane
 - controlplaneexposure
+- controlplanebackup
```

```suggestion
- etcdbackup
```

To make it more precise what it is used for

---

As discussed, we should rather keep the `controlplane` prefix for these webhooks as they are referred to as "controlplane" webhooks in several places.